include/aspect/particle/property/interface.h

```diff
@@ -59,6 +61,31 @@ namespace aspect
       };
 
       /**
+       * This enum controls how to initialize the properties of particles that
+       * have been added later than the initial particle creation, e.g. to
+       * improve the load balance or to prevent empty cells.
+       */
+      enum InitializeLateParticles
```

Comment:
This enum is awkwardly named because its first word is a verb -- but it  doesn't actually *do* anything. Maybe call it `InitializationModeForLateParticles`? This would also suggest renaming the `late_initialize_mode` function to `late_initialization_mode`.